tsp/entry.py

```diff
@@ -43,9 +56,29 @@ class EntryHeader:
 
     def __init__(self, params):
         '''
-        Displayed name
+        Constructor
         '''
-        self.name = params
+
+        # Name for this header.
+        if HEADER_NAME_KEY in params:
+            self.name = params.get(HEADER_NAME_KEY)
+            del params[HEADER_NAME_KEY]
+        else:
+            self.name = None
+
+        # Tooltip for this header.
+        if HEADER_TOOLTIP_KEY in params:
+            self.tooltip = params.get(HEADER_TOOLTIP_KEY)
+            del params[HEADER_TOOLTIP_KEY]
+        else:
+            self.tooltip = None
+
+        # Data type for this header.
+        if HEADER_DATA_TYPE_KEY in params:
+            self.data_type = EntryHeaderDataType(params.get(HEADER_DATA_TYPE_KEY))
+            del params[HEADER_DATA_TYPE_KEY]
+        else:
+            self.data_type = None
```

Comment:
It could be an argument to put this as `EntryHeaderDataType.STRING` since it is the specified fallback in the absense of the "dataType" field. I put it to `None` to be transparent about what the server returns.

Reply:
That's ok, I think. Calling methods can default to String